package.json

```diff
@@ -188,5 +188,44 @@
     "which": "^1.3.0",
     "winreg": "^1.2.4",
     "xterm": "^3.0.2"
+  },
+  "jupyterlab": {
```

Comment:
Do you end up using this data embedded in the `package.json` file as part of the extension loading system?

Reply:
This data is used to iterate over when it is determined which extensions have which status. This is copied from the main JupyterLab repo. I wanted to use this list to import the extensions, but this kind of dynamic importing does not appear to be possible. Anyway, this is redundant right now since the extensions are also listed in another file. I'll remove it.